themes/finna2/templates/RecordDriver/DefaultRecord/data-termsOfUse.phtml

```diff
@@ -2,10 +2,19 @@
 <?php foreach ($data as $field): ?>
   <?=!empty($field['material']) ? $this->escapeHtml($field['material']) . ': ' : '' ?>
   <?=!empty($field['terms']) ? $this->escapeHtml($field['terms']) : '' ?>
+  <?=!empty($field['rightsSource']) ? $this->escapeHtml($this->transEscWithPrefix('access_restriction_source_' . $field['rightsSource'])) : '' ?>
```

Comment:
transEscWithPrefix:n kanssa ei escapeHtml:ää enää. Ja prefix ja käännös erillisinä parametreina: `$this->transEscWithPrefix('access_restriction_source_', $field['rightsSource'])`